src/Internal.AspNetCore.Sdk/build/Common.targets

```diff
@@ -10,16 +10,72 @@ For single-tfm projects, this will be imported from build/Internal.AspNetCore.Sd
 -->
 <Project>
 
-  <!-- workaround https://github.com/NuGet/Home/issues/4726 -->
-  <PropertyGroup>
-    <IncludeSymbols Condition="'$(NuspecFile)'!=''">false</IncludeSymbols>
-  </PropertyGroup>
-
   <PropertyGroup Condition=" '$(IncludeSymbols)' != 'false' ">
     <GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);_EnsureDebugTypeIsPortable</GenerateNuspecDependsOn>
+    <PackDependsOn Condition="'$(NuspecFile)' != ''">$(PackDependsOn);_GenerateSymbolsNupkg</PackDependsOn>
+
+    <!-- workaround https://github.com/NuGet/Home/issues/4726 -->
+    <IncludeSymbols Condition="'$(NuspecFile)'!=''">false</IncludeSymbols>
+    <IncludeSource Condition="'$(NuspecFile)'!=''">false</IncludeSource>
+
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);_EnsureDebugTypeIsPortable</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
 
+  <Target Name="_GenerateSymbolsNupkg" >
+    <Error Condition="'$(NuspecFile)' != '' AND '$(SymbolsNuspecFile)' == ''"
+      Text="NuspecFile was specified but SymbolsNuspecFile was not specified."
+      Code="KRB5006" />
+
+    <ConvertToAbsolutePath Paths="$(SymbolsNuspecFile)">
+      <Output TaskParameter="AbsolutePaths" PropertyName="SymbolNuspecFileAbsolutePath" />
+    </ConvertToAbsolutePath>
+
+    <ConvertToAbsolutePath Paths="$(PackageOutputPath)symbols-nupkg\">
```

Comment:
I think it would be better to make this `$(IntermediateOutputPath)symbols-nupkg\`.